### DISCUSSION TITLE
How can I retrieve my okta token?

### DISCUSSION THREAD

**Discussion by Pixelatex** [nextauthjs/next-auth#1985] [Help]
## Question 💬

I am using nextauth to connect to OKTA. However, I need to use my okta token later on in my API and I need to verify if the token is genuine.

How can I do this? Because it seems like nextauth signs a jwt token itself so I can no longer uses okta's verify package to confirm its validity.

So I basically want to get the original OKTA token instead of the nextauth signed jwt token.

## How to reproduce ☕️

n/a

## Contributing 🙌🏽

It takes a lot of work 🏋🏻‍♀️ maintaining a library like `next-auth`; any contribution is more than welcome 💚

In case you're willing to help answer this question, please let us know here, and we'll reach you 😊 . Otherwise, you can have a look at the issues labelled with [`"good first issue"`](https://github.com/nextauthjs/next-auth/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and pick any of them.

**Comment by balazsorban44** [comment]
Check the jwt, signIn and session callbacks: https://next-auth.js.org/configuration/callbacks

Your okta token should be on the `account` parameter somewhere if I had to guess. you can do whatever you want with it.
